imgix.gemspec

```diff
@@ -20,4 +20,6 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 1.9.0'
   spec.add_dependency 'addressable'
+  spec.add_dependency 'http'
```

Comment:
sorry if I'm being dense, but is this gem actually being used here? AFAIK, it uses an `HTTP` class, and this PR only uses `Net::HTTP`, not `HTTP`. It'd be great to not require it if we don't really need it!

Reply:
@benhutton You are absolutely correct. Addressed in #38. Thanks for catching that.